pkg/operator/ceph/controller/spec.go

```diff
@@ -825,6 +825,37 @@ func LogCollectorContainer(daemonID, ns string, c cephv1.ClusterSpec) *v1.Contai
 	}
 }
 
+// rgw operations will be logged in sidecar ops-log
+func RgwOpsLogContainer(daemonID, ns string, c cephv1.ClusterSpec, Resources v1.ResourceRequirements) *v1.Container {
+
+	// Generate ops log file name based on daemon ID
+	opsLogFile := fmt.Sprintf("ops-log-%s", daemonID)
+	cmdLogRotate := "true"
+
+	if c.LogCollector.Enabled {
+		maxLogSize, periodicity := GetLogRotateConfig(c)
+		cmdLogRotate = fmt.Sprintf(cronLogRotate, opsLogFile, periodicity, maxLogSize.String(), "7")
+	}
+
+	return &v1.Container{
+		Name: "ops-log",
+		Command: []string{
+			"bash",
+			"-x", // Enable debugging mode
+			"-m", // Enable job control
```

Comment:
What is the `-m` argument? `man bash` doesn't list this. Is this maybe a typo or vestigial from a `python -m` call?

Reply:
it's for the job control, but i can skip it if we have another container for the log process 
```
              -m      Monitor mode.  Job control is enabled.  This option is on by default for interactive shells on systems that
                      support it (see JOB CONTROL above).  All processes run in a separate process group.  When a background job
                      completes, the shell prints a line containing its exit status.
```